app/controllers/concerns/kinetic_cafe/error_handler.rb

```diff
@@ -101,16 +110,22 @@ def kinetic_cafe_error_render_json(error)
     end
   end
 
+  def kinetic_cafe_error_handle_post_error(_error)
+  end
+
   # Write the provided error to the Rails log using the value of
   # #kinetic_cafe_error_handler_log_locale. If the error has a cause, log
   # that as well.
```

Comment:
Put a bit of documentation about expecting `Rails.logger.class` to have constants matching `error.severity.upcase` to describe the log-level-severity.